src/plugins/presentation_util/public/hooks/use_load.ts

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Don't think this file is used anymore, so it should be removed.

Reply:
Yes, you are right, I've forgotten to remove it.